ci-operator/step-registry/ipi/conf/aws/proxy/ipi-conf-aws-proxy-commands.sh

```diff
@@ -219,10 +203,10 @@ Resources:
   ProxyInstance:
     Type: AWS::EC2::Instance
     Properties:
-      ImageId: !Ref RhcosAmi
+      ImageId: !Ref Ami
       IamInstanceProfile: !Ref ProxyInstanceProfile
       KeyName: "openshift-dev"
-      InstanceType: "i3.large"
+      InstanceType: "m5.xlarge"
```

Comment:
Can you motivate this change in the commit message?

Reply:
I'll try again without it, AFAICS `i3.large` does boot fine but I was trying to use one of our standard instance types.

Reply:
```console
$ git --no-pager log --oneline -p -1 origin/pr/11750 | grep 'Port\|InstanceType'
634798fce4 steps/proxy: Port to Fedora CoreOS
-      InstanceType: "i3.large"
+      InstanceType: "m5.xlarge"
```

Looks like you're still using m5.xlarge with your current tip, but still aren't motivating the change in the commit message?  I'm fine with either:

* leave this as i3.large or
* commit message says:

    > Move to an m5 instance type, because we prefer m4/m5 for cluster instances, and it's nice to have a familiar type.

    or some such.